src/Controller/SponsorshipLevelBenefit/Edit.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\SponsorshipLevelBenefit;
+
+use App\Service\DTO\SponsorshipLevelBenefitData;
+use App\Service\FormatSponsorshipLevelBenefit;
+use App\Service\Manager\SponsorshipLevelManagerInterface;
+use App\Service\Manager\SponsorshipLevelBenefitManagerInterface;
+use App\Service\Manager\SponsorshipBenefitManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Csrf\CsrfToken;
+use Symfony\Component\Security\Csrf\CsrfTokenManagerInterface;
+use Twig_Environment;
+
+final class Edit extends Controller
```

Comment:
Do not extend `Controller` or `AbstractController`, it comes with a service locator and more awful things, so please try to avoid it